pkg/scalers/metrics_api_scaler.go

```diff
@@ -134,8 +134,8 @@ func parseMetricsAPIMetadata(config *ScalerConfig) (*metricsAPIScalerMetadata, e
 		meta.method = "header"
 		meta.enableAPIKeyAuth = true
 
-		if config.TriggerMetadata["method"] == "query" {
-			meta.method = "query"
+		if config.TriggerMetadata["method"] == promQuery {
```

Comment:
It is not ideal to reference const from another scaler, we should introduce a one specific for this scaler instead